package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "devicon",
-  "version": "2.15.0",
+  "version": "2.15.1",
```

Comment:
Please don't update the package-lock and package files :)

Looks like it might have happened because you based your branch on master instead of develop. Try checking out develop instead, and cherry-picking this commit: https://github.com/devicons/devicon/pull/1438/commits/a28d5ad0f21ceae5ed56ac6f697f3141ae5cc465